### PR TITLE
toggle change

### PR DESCRIPTION
**Pull request by Ninty9**
kept the onEnable and onDisable functions for eventual feedback functionality in the future

### CODE REVIEW

File: src/main/java/com/domain/redstonetools/features/toggleable/ToggleableFeature.java [hunk 1]
```diff
@@ -17,26 +17,17 @@ protected void registerCommands(CommandDispatcher<ServerCommandSource> dispatche
...
+                .executes(this::run));
     }
 
     public boolean isEnabled() {
         return enabled;
     }
 
-    private int enable(CommandContext<ServerCommandSource> context) throws CommandSyntaxException {
-        enabled = true;
+    private int run(CommandContext<ServerCommandSource> context) throws CommandSyntaxException {
```
Comment (Matthias1590) [inline src/main/java/com/domain/redstonetools/features/toggleable/ToggleableFeature.java:27]: might be nice to rename this to toggle

**Comment by Matthias1590** [comment]
might be nice to just send feedback in the run method so features don't have to do that themselves

**Review by Matthias1590 — CHANGES_REQUESTED** [review]

**Comment by Ninty9** [comment]
> might be nice to just send feedback in the run method so features don't have to do that themselves

that's definitely doable, but shouldn't we wait until we have the feedback util for that?

**Comment by Matthias1590** [comment]
> > might be nice to just send feedback in the run method so features don't have to do that themselves
> 
> that's definitely doable, but shouldn't we wait until we have the feedback util for that?

Might be easier to just do it now so we can refactor it later and not risk forgetting about it

**Comment by Ninty9** [comment]
That should be everything

**Review by Matthias1590 — APPROVED** [review]

**Pull request merged**